controllers/storagecluster/cephcluster.go

```diff
@@ -208,6 +209,16 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		}
 	}
 
+	if sc.Spec.HostNetwork && sc.Spec.Network != nil {
+		if sc.Spec.Network.AddressRanges != nil && sc.Spec.Network.AddressRanges.Public != nil {
```

Comment:
Please add a comment about the change  and combine all thecks into one if and do we need `sc.Spec.HostNetwork` network check as well?

Reply:
yes sc.Spec.HostNetwork check is required as AddressRanges can also be used in case of multus